packages/charts/react-charting/src/utilities/image-export-utils.ts

```diff
@@ -63,11 +65,15 @@ function toSVG(chartContainer: HTMLElement, background: string) {
   }
 
   const { width: svgWidth, height: svgHeight } = svg.getBoundingClientRect();
-  const legendGroup = cloneLegendsToSVG(chartContainer, svgWidth, svgHeight);
+  const legendGroup =
+    typeof legendsToSvgCallback === 'function'
+      ? legendsToSvgCallback(svgWidth, isRTL)
```

Comment:
do we need to pass isRTL flag everywhere. We can get it directly from utils getRTL?

Reply:
Since we wanted to use the same util in v9, and the layout direction can only be accessed using the useFluent hook there, I retrieved it inside the chart and passed it to the util. Also, it seemed like a better design to manage these configs at the chart level and pass them down to the utils.

Reply:
The direction is not controlled by the chart but the outer environment hosting the chart. Can we call the hook without using a functional component.

Reply:
No, hooks should only be called from react function components or custom hooks. [Rules of Hooks - React](https://react.dev/reference/rules/rules-of-hooks)